src/Umbraco.Core/Models/Rdbms/AuditEntryDto.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Umbraco.Core.Persistence;
+using Umbraco.Core.Persistence.DatabaseAnnotations;
+using Umbraco.Core.Persistence.DatabaseModelDefinitions;
+
+namespace Umbraco.Core.Models.Rdbms
+{
+    [TableName(TableName)]
+    [PrimaryKey("id")]
+    [ExplicitColumns]
+    internal class AuditEntryDto
+    {
+        public const string TableName = "umbracoAudit";
+
+        [Column("id")]
+        [PrimaryKeyColumn]
+        public int Id { get; set; }
+
+        [Column("performingUserId")]
+        public int PerformingUserId { get; set; }
```

Comment:
Shouldn't this have a foreign key?

Reply:
idea was we could delete a user yet want to keep all auditing data, just in case - will add a comment